### PR TITLE
Feature/api version

### PR DESCRIPTION
**Pull request by erikeelde**
Half of this PR is in https://github.com/iZettle/wrench/pull/40. It's based on it.

### CODE REVIEW

**Comment by joezorry** [comment]
@erikeelde If this gets merge then there will be duplicated commits right?

**Comment by erikeelde** [comment]
Should be the same commits so we should be good I think.. The important thing would be to rebase agains it's base-branch though.

File: wrench-service/build.gradle [hunk 1]
```diff
@@ -32,6 +32,7 @@ dependencies {
     implementation 'com.android.support:support-annotations:27.0.1'
 
     testImplementation 'junit:junit:4.12'
+    compile "org.jetbrains.kotlin:kotlin-stdlib-jre7:$kotlin_version"
```
Comment (joezorry) [inline wrench-service/build.gradle]: api or implementation?

File: build.gradle [hunk 2]
```diff
@@ -1,13 +1,13 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 
 buildscript {
-    ext.kotlin_version = '1.1.51'
+    ext.kotlin_version = '1.1.60'
```
Comment (joezorry) [inline build.gradle]: If you change this, you might as well bump to 2.0.

File: wrench-service/build.gradle (hunk 1 above)
Comment (erikeelde) [reply wrench-service/build.gradle]: Should it be added in the first place?

File: build.gradle (hunk 2 above)
Comment (erikeelde) [reply build.gradle]: Souns like a bigger bump that wareants a separate pr tbh. :)

Comment (joezorry) [reply build.gradle]: 1.2.... The previous comment never happened....

Comment (joezorry) [reply build.gradle]: Trust AS
😁

**Pull request merged**